### PR TITLE
Azure Advisor Cost Optimization Workbook - November Update

### PR DESCRIPTION
**Pull request by arthurclares**
## PR Checklist

* [ ] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.
This is the November update for Advisor Cost Optimization Workbook. This month's changes include:

- New subtab for RI and Savings Plan
- Savings Plan recommendations
- Reservations recommendations
- AHB for Stack HCI
- New Synapse query under the Top 10 Services tab
- New Log Analytics query under the Top 10 Services tab
- Moved AKS and App services query under the Top 10 Services tab

* [x] Validate your changes using one or more of the [testing](../Documentation/Testing.md) methods.

### If adding or updating templates:
* [ ] post a screenshot of templates and/or gallery changes
* [ ] ensure your template has a corresponding gallery entry in the gallery folder
* [ ] If you are adding a new template, add your team and template/gallery file(s) to the CODEOWNERS file. CODEOWNERS entries should be teams, not individuals
* [ ] ensure all steps have meaningful names
* [ ] ensure all parameters and grid columns have display names set so they can be localized
* [ ] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [ ] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [ ] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [ ] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

### CODE REVIEW

File: Workbooks/Azure Private MEC/PMEC Site Overview/PMEC Site Overview.workbook
Comment (pkrishnanms) [inline Workbooks/Azure Private MEC/PMEC Site Overview/PMEC Site Overview.workbook:1]: is this file required for this PR? this is not under Advisor folder

Comment (arthurclares) [reply Workbooks/Azure Private MEC/PMEC Site Overview/PMEC Site Overview.workbook:1]: @pkrishnanms It is no required, it was submitted by mistake.

**Comment by pkrishnanms** [comment]
please resolve conflicts and remove this file change from this PR instead of deleting it in this PR
